pkg/controller/controller_instance.go

```diff
@@ -519,8 +519,9 @@ func (c *controller) reconcileServiceInstance(instance *v1beta1.ServiceInstance)
 	// osb client handles whether or not to really send this based
 	// on the version of the client.
 	requestContext := map[string]interface{}{
-		"platform":  ContextProfilePlatformKubernetes,
-		"namespace": instance.Namespace,
+		"platform":   ContextProfilePlatformKubernetes,
+		"namespace":  instance.Namespace,
+		"cluster-id": c.clusterID,
```

Comment:
Only set this field if feature gate is active

Reply:
It's likely that this feature will go through a phase where it uses an alpha API, so we need a feature gate that lets you opt into this if you specifically want it. 

> Plus, when done, I don't think we should require people to ask for this info to be included in the "context" - it should "just be there" the same way namespace is "just there".

When the feature is GA, sure.  Until then we need a feature gate.

Reply:
Which alpha API are you thinking of? Or is that part of our chat later today?

Reply:
That's what I wanted to talk about later today :)

Reply:
change it to `clusterID` to be consistent with the rest of svc-cat's fields.